### DISCUSSION TITLE
Question about ptp role master in AVD between uplinks spine/leaf

### DISCUSSION THREAD

**Discussion by geckigek** [aristanetworks/avd#4674] [Q&A]
Hi there,

we noticed today a PTP problem on some tests with PTP in combination with a test regarding network transitions. PTP i salso running in the other network segment, which is connected to a border leaf of the AVD fabric. The other segment has a higher PTP priority set as the grandmaster of the avd media fabric. The grandmaster of the avd fabric is connected to the spine. 

It happend that the spine synced itself to the ptp grandmaster from the other network segment. Surely we could disable the ptp on the connection between the border leaf and the other network segment, but we noticed that in our setup AVD is not implementing a PTP security feature which would be nice to have. The security in that specific case regarding PTP would be that the downlink ports of spine are automaticly configured as ptp role master when ptp is enabled in the whole fabric.

Is there a way to implement such a feature with the current version of avd in the eos_design role or could it be a possible feature request on github to implement this?

Best regards 

Stefan

**Comment by ClausHolbechArista** [comment]
We have not forgotten about this question. Still investigating, since I believe it would be a bad idea leading to more widespread issues if you implement this.
